bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -174,27 +176,31 @@
 		</state>
 	</channel-type>
 
-	<channel-type id="timestamp" advanced="true">
+	<channel-type id="timestamp">
 		<item-type>DateTime</item-type>
-		<label>@text/timestampLabel</label>
-		<description>@text/timestampDescription</description>
-		<category>Observation time</category>
-		<state readOnly="true"></state>
+		<label>@text/timestampChannelLabel</label>
+		<description>@text/timestampChannelDescription</description>
```

Comment:
timestampChannelLabel and timestampChannelDescription are not defined in the default properties file.

Reply:
Done.